README.md

```diff
@@ -1,13 +1,15 @@
 # YIPs [![Discord](https://img.shields.io/discord/734804446353031319.svg?color=768AD4&label=discord&logo=https%3A%2F%2Fdiscordapp.com%2Fassets%2F8c9701b98ad4372b58f13fd9f65f966e.svg)](https://discordapp.com/channels/734804446353031319/) [![Telegram](https://img.shields.io/badge/chat-on%20Telegram-blue.svg)](https://t.me/yearnfinance) [![Twitter Follow](https://img.shields.io/twitter/follow/iearnfinance.svg?label=iearnfinance&style=social)](https://twitter.com/iearnfinance)
 
-yEarn Improvement Proposals (YIPs) describe standards for the yEarn platform, including core protocol specifications, client APIs, and contract standards.
- 
+YEarn Improvement Proposals (YIPs) describe standards for the yEarn platform, including core protocol specifications, client APIs, and contract standards.
+
+![Validate](https://github.com/sambacha/YIPS/workflows/Validate/badge.svg)
+
 ## Contributing
 
- 1. Review [YIP-0](YIPS/yip-0.md).
- 2. Fork the repository by clicking "Fork" in the top right.
- 3. Add your YIP to your fork of the repository. There is a [template YIP here](yip-X.md).
- 4. Submit a Pull Request to yEarn's [YIPs repository](https://github.com/iearn-finance/YIPS/).
+1.  Review [YIP-0](YIPS/yip-0.md).
+2.  Fork the repository by clicking "Fork" in the top right.
+3.  Add your YIP to your fork of the repository. There is a [template YIP here](yip-X.md).
+4.  Submit a Pull Request to yEarn's [YIPs repository](https://github.com/iearn-finance/YIPS/).
```

Comment:
`Yearn` and all other occurrences, please

Reply:
\-\[x\] completed